internal/corazawaf/transaction.go

```diff
@@ -104,6 +104,8 @@ type Transaction struct {
 	audit bool
 
 	variables TransactionVariables
+
+	transformationCache map[transformationKey]transformationValue
```

Comment:
@M4tteoP Though besides that reusing maps is also important, adding to the transaction which is already pooled made it pretty simple